onnxruntime/core/providers/nnapi/nnapi_builtin/nnapi_lib/readme.txt

```diff
@@ -0,0 +1,19 @@
+1. Files: nnapi_implementation.h
+          nnapi_implementation.cc
+          NeuralNetworksTypes.h
+These files are copied from TensorFlow Lite project
+https://github.com/tensorflow/tensorflow/tree/master/tensorflow/lite/nnapi
+
```

Comment:
Do we need an addition for TF Lite in ThirdPartyNotices.txt? Not sure if it's covered by the existing TF notice.

Reply:
TF Lite use same license as Tensor Flow, the one in the ThirdPartyNotices.txt is more recent